src/test/kotlin/org/rust/ide/intentions/RsIntentionTestBase.kt

```diff
@@ -11,9 +11,27 @@ import com.intellij.util.ui.UIUtil
 import org.intellij.lang.annotations.Language
 import org.rust.RsTestBase
 import org.rust.fileTreeFromText
+import java.nio.file.Paths
 
 abstract class RsIntentionTestBase(val intention: IntentionAction) : RsTestBase() {
 
+    fun testIntentionHasDocumentation() {
```

Comment:
Let's check it only if `intention` is inheritor of `RsElementBaseIntentionAction`. We should check documentation only of our intentions. But tests can invoke any intention (see `org.rust.ide.intentions.InjectLanguageIntentionTest`)